components/form.css

```diff
@@ -0,0 +1,82 @@
+.form{
+    width: 100%;
+}
+
+.label{
+    font-size: 1.25rem;
+    letter-spacing: initial;
+}
+
+.input{
+    margin: 10px auto;
+    border-radius: 4px;
+    outline: none;
+    border: 1px solid var(--gray-700);
+    padding: 10px 1rem;
```

Comment:
1rem = 10px , so why not use 1rem for both?

Reply:
because 1rem = 16px. Thanks for pointing out anyways, I have changed the units to multiples of 8.

Reply:
1rem is 16px right? Going by root browser font size